actions/qemuimg2disk/v1/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -o xtrace
+qemu-img convert ${IMG_URL:?} -O ${FORMAT:-host_device} ${DEST_DISK:?} ${EXTRA_ARGS}
```

Comment:
TIL: `qemu-img` supports HTTP(s).

Reply:
It uses libcurl so it can do what curl can do!